components/automate-ui/src/app/services/event-feed/event-feed.service.ts

```diff
@@ -230,6 +233,12 @@ export class EventFeedService {
       filters = concat(filters, {type: ENTITY_TYPE_TAG, text: ENTITY_TYPE_COOKBOOK_VERSION_TAG});
     }
 
+    if (includes(ENTITY_TYPE_COOKBOOK_TAG, entityTypes) &&
+      !includes(ENTITY_TYPE_ARTIFACT_COOKBOOK_VERSION_TAG, entityTypes)) {
+      filters = concat(filters,
+        {type: ENTITY_TYPE_TAG, text: ENTITY_TYPE_ARTIFACT_COOKBOOK_VERSION_TAG});
+    }
+
```

Comment:
When the "cookbook" filter is added also add the "cookbook_artifact_version" filter.